packages/paste-core/components/combobox/src/multiselect/MultiselectCombobox.tsx

```diff
@@ -0,0 +1,314 @@
+        }
+
+        // FIX: Looping fix, scroll to the beginning of the list when the first item is highlighted
+        if (node && highlightedIndex === 0) {
+          rowVirtualizer.scrollToIndex(0);
+        }
+        // FIX: Looping fix, scroll to the end of the list when the last item is highlighted
+        if (node && highlightedIndex === size - 1) {
+          rowVirtualizer.scrollToIndex(size - 1);
+        }
+      },
+      [items, selectedItems, highlightedIndex]
+    );
+
+    // FIX: a11y issue where `aria-expanded` isn't being set until the dropdown opens the very first time
+    const comboboxProps = getComboboxProps({
+      disabled,
+      role: 'combobox',
+    });
+    const ariaExpanded = comboboxProps['aria-expanded'] || 'false';
+
+    return (
+      <Box position="relative" element={`${element}_WRAPPER`}>
+        <Label disabled={disabled} required={required} variant={variant} {...getLabelProps()}>
+          {labelText}
+        </Label>
+        <InputBox
+          disabled={disabled}
+          hasError={hasError}
+          element={element}
+          insertBefore={insertBefore}
+          insertAfter={insertAfter}
+          variant={variant}
+        >
+          <Box
+            {...getToggleButtonProps({disabled})}
+            {...comboboxProps}
+            aria-expanded={ariaExpanded}
+            position="relative"
+            width="100%"
+            paddingY="space30"
+            paddingX="space40"
+          >
+            <FormPillGroup
+              {...pillState}
+              aria-label="Products:"
+              element={`${element}_PILL_GROUP`}
+              display="inline-flex"
+            >
+              {selectedItems.map(function renderSelectedItems(selectedItemPill: Item, index: number) {
+                return (
+                  <FormPill
+                    variant={hasError ? 'error' : 'default'}
+                    element={`${element}_PILL`}
+                    key={`${itemToString(selectedItemPill)}-${index}`}
+                    onDismiss={() => {
+                      removeSelectedItem(selectedItemPill);
+                    }}
+                    // Prevents focus from being snatched from the pill to the input
+                    onSelect={(event: React.MouseEvent<HTMLButtonElement>) => event.stopPropagation()}
+                    {...pillState}
+                  >
+                    {itemToString(selectedItemPill)}
+                  </FormPill>
+                );
+              })}
+            </FormPillGroup>
+            <GrowingInput
+              // we spread props into `getInputProps` so that Downshift handles events correctly
+              {...getInputProps({
+                ...getDropdownProps({ref, preventKeyAction: isOpen}),
+                disabled,
+                required,
+                ...props,
+              })}
+              aria-describedby={helpTextId}
+              element={`${element}_ELEMENT`}
```

Comment:
Issue: the element should be `${element}_INPUT`

Reply:
I agree, but left it as ELEMENT to match Combobox :/